recipes/vnu-validator/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "23.4.11" %}
+
+package:
+  name: vnu-validator
+  version: {{ version }}
+
+source:
+  # building from source is... untenable due to sh/py/ant/java/git sandwich
+  url: https://registry.npmjs.org/vnu-jar/-/vnu-jar-{{ version }}.tgz
+  sha256: cbba595a86d680d48e9c5261a16d9ef409e5658ad8aa47081c593690a652cf0f
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - __{{ noarch_platform }}
+    - openjdk >=8
+
+test:
+  requires:
+    - m2-grep  # [win]
+    - python
+    - pytest
+
+about:
+  home: https://validator.github.io/validator
+  license: MIT
+  # TODO: see if licenses are in jar (unlikely)
+  license_file: LICENSE
```

At least the git repo has something for thirdparties. 
```suggestion
  license_file: 
    - LICENSE
    - CREDITS
```